src/rust/ide/view/graph-editor/src/component/visualization/container/action_bar.rs

```diff
@@ -86,6 +227,9 @@ ensogl::define_endpoints! {
         visualisation_selection (Option<visualization::Path>),
         mouse_over              (),
         mouse_out               (),
+        action_reset_position   (),
```

Comment:
rename that to `on_container_reset_position`, and the other thing to `container_drag_state`. The first is a convention we use in new components. The second is more readable for e but Im open for suggestions to immprove it even further.